### PR TITLE
Adding hostNetwork flag

### PR DESCRIPTION
**Pull request by etiennnr**
**What this PR does / why we need it**:
Adding the hostNetwork flag (false by default, true if flag is present) as this is very useful when troubleshooting a network issue

**Which issue(s) this PR fixes**:
Fixes # N/A

**Special notes for your reviewer**:

**Release note**:

```breaking operator
The `hacks/ops-pod` script defaults `hostNetwork` of the ops-pod to `false`. You can enable it using the flag `-o` or `--hostnetwork`
```

### CODE REVIEW

**Review by petersutter — APPROVED** [review]
/lgtm

I have also updated the release notes (added breaking change)

**Comment by etiennnr** [comment]
@petersutter I unfortunately don't have the write access on the repo. What's the normal time between approve and merge? Do you need to wait for someone else to merge?

**Comment by petersutter** [comment]
@plkokanov do you also want to take a look?

File: hacks/ops-pod
Comment (plkokanov) [inline hacks/ops-pod:53]:
```suggestion
hostnetwork=${FALSE}

File: hacks/ops-pod
Comment (plkokanov) [inline hacks/ops-pod:92]:
```suggestion
      hostnetwork=${TRUE}

**Review by plkokanov — CHANGES_REQUESTED** [review]
Just some minor comments to align the new boolean var with the already existing ones.

File: hacks/ops-pod
Comment (petersutter) [reply hacks/ops-pod:53]: FALSE variable is a number, however the value of `hostnetwork` will then be used to set `spec.template.spec.hostNetwork`, which expects a `bool` value -> we can't do this change

File: hacks/ops-pod
Comment (petersutter) [reply hacks/ops-pod:92]: see https://github.com/gardener/ops-toolbelt/pull/86#discussion_r960306405

File: hacks/ops-pod
Comment (etiennnr) [reply hacks/ops-pod:53]: Indeed, I tested it like this before and it didn't work...

I could add like an IF to check for that variable and then assign the associated string, but I think that it's easier and simpler (thus better) to just use the string values directly.

File: hacks/ops-pod
Comment (etiennnr) [reply hacks/ops-pod:92]: same as above

File: hacks/ops-pod
Comment (plkokanov) [reply hacks/ops-pod:53]: Let's leave it as it is. Was a bit distracted when I did the review sorry.

**Review by plkokanov — APPROVED** [review]

**Pull request merged**
